Cargo.toml

```diff
@@ -20,34 +20,26 @@ name = "cosmian_findex"
 path = "src/lib.rs"
 
 [features]
-ffi = ["interfaces", "futures", "interfaces", "serde", "serde_json"]
-sqlite = ["rusqlite", "faker_rand", "interfaces", "serde", "serde_json"]
-wasm_bindgen = [
-  "interfaces",
-  "js-sys",
-  "wasm-bindgen",
-  "wasm-bindgen-test",
-  "futures",
-  "web-sys",
-  "wasm-bindgen-futures",
-  "interfaces",
-]
+ffi = ["interfaces", "futures", "interfaces"]
+sqlite = ["rusqlite", "faker_rand", "interfaces"]
+wasm_bindgen = ["interfaces", "js-sys", "wasm-bindgen", "wasm-bindgen-test", "futures", "web-sys", "wasm-bindgen-futures", "interfaces"]
 interfaces = []
 python = ["interfaces", "pyo3", "futures"]
 
 [dependencies]
 async-recursion = "1.0.0"
 base64 = "0.13"
-cosmian_crypto_core = "5.0"
+cosmian_crypto_core = "6.0"
 faker_rand = { version = "0.1", optional = true }
 futures = { version = "0.3", optional = true }
+hex = "0.4.3"
 js-sys = { version = "0.3", optional = true }
 leb128 = "0.2"
 pyo3 = { version = "0.17", features = ["extension-module", "abi3", "abi3-py37", "generate-import-lib"], optional = true }
 rand = "0.8"
 rusqlite = { version = "0.28", features = ["bundled"], optional = true }
-serde = { version = "1.0", features = ["derive"], optional = true }
-serde_json = { version = "1.0", optional = true }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
```

Comment:
Are you sure you can't keep them optional ? There should be no serialization without the `interfaces` feature.

Reply:
fixed.